### PR TITLE
Add react-native-maps

### PR DESCRIPTION
**Pull request by tbergq**
---
Please check this before merging (do not delete this checklist):

- [ ] it works in landscape and portrait mode
- [ ] it uses `cacheConfig.force=true` if offline access doesn't make sense

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,8 @@
 
 ```
 yarn install
+cd ios
+pod install
```
Comment (jaroslav-kubicek) [inline README.md]: `brew install cocoapods` might be necessary as well

File: ios/Podfile [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
```
[the remark this answers is not in the document]
Comment (tbergq) [reply ios/Podfile]: https://cocoapods.org/

File: package.json [hunk 3]
```diff
@@ -17,8 +17,8 @@
     "postinstall": "node ./scripts/configureApplication.js"
   },
   "dependencies": {
-    "react": "16.2.0",
-    "react-native": "0.52.0"
+    "react": "16.0.0",
```
[the remark this answers is not in the document]
Comment (tbergq) [reply package.json]: v. 0.52.0 is shipped with a bug that prevents build when we add Pods
https://github.com/facebook/react-native/issues/17274

[the remark this answers is not in the document]
Comment (jaroslav-kubicek) [reply package.json]: nice, thanks, sad json has no support for comments :-/

File: README.md (hunk 1 above)
Comment (tbergq) [reply README.md]: `brew install node` and `npm i -g yarn` I guess

File: package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (mrtnzlml) [reply package.json]: Haha, yes - I don't know who started to use JSONs for config files. Config files NEED comments (we are configuring dependencies here). Biggest fail for modern applications... :))

[the remark this answers is not in the document]
Comment (tbergq) [reply package.json]: Let's hope comments can be added to JSON files some time in the future

File: .gitignore [hunk 4]
```diff
@@ -25,6 +25,8 @@ DerivedData
 *.ipa
 *.xcuserstate
 project.xcworkspace
+ios/Pods
+Podfile.lock
```
Comment (mrtnzlml) [inline .gitignore]: How does the `Podfile.lock` work? Is it the same as for example `yarn.lock`? In this case we should probably commit it.

**Comment by mrtnzlml** [comment]
@tbergq Integrated, thanks. Also, could you please install Android Studio and always verify that it works? It was not ready for Android... 🙂

File: .gitignore (hunk 4 above)
Comment (tbergq) [reply .gitignore]: I saw in the repo for the native iOS app, their gitignore contained Podfile.lock. That is why I added it

Comment (mrtnzlml) [reply .gitignore]: Ok, this comment is all I need to know... 😊

**Comment by tbergq** [comment]
I have android studio installed, an it ran. What did you have to do to make it run?

**Comment by mrtnzlml** [comment]
https://github.com/kiwicom/react-native-app/commit/4c9201fd5165dec99a6439ae2a2548025fd68ded#diff-f43dd382b06e67a294bf91c521e24d5dR29

I hope it's now wrong - but it didn't work without it and it's part of the installation guidelines...

**Pull request closed without merging**